devServer.js

```diff
@@ -13,7 +13,7 @@ app.use(require('webpack-dev-middleware')(compiler, {
 
 app.use(require('webpack-hot-middleware')(compiler));
 
-app.get('/', function(req, res) {
```

Comment:
Odd, it was broken for both Sanjuro and me.
On Fri, Apr 15, 2016 at 5:54 PM David Daniel <notifications@github.com>
wrote:

> In devServer.js
> <https://github.com/hackoregon/btc-frontend-2016-ReactRedux/pull/31#discussion_r59958921>
> :
>
> > @@ -13,7 +13,7 @@ app.use(require('webpack-dev-middleware')(compiler, {
> >
> >  app.use(require('webpack-hot-middleware')(compiler));
> >
> > -app.get('/', function(req, res) {
>
> Jason made this change.. but it's working for me without merging this in
> right now..
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hackoregon/btc-frontend-2016-ReactRedux/pull/31/files/3803c7ca23f49e8a6fac025576a2557a286ba73c#r59958921>
>